src/wildcard.cpp

```diff
@@ -30,7 +30,7 @@ void wildcard_expand(const std::shared_ptr<char> arg)
   char * a = arg.get();
 
   for (int i = 0; *(a + 1); a++, i++) {
-    if ((*a == '/' && *(a + 1) == '.' && *(a + 1) == '*') ||
+    if ((*a == '/' && (*(a + 1) == '.' || *(a + 1) == '*')) ||
```

Comment:
This might be the root cause for travis CI failure

Reply:
it isn't; this piece of code is resolving hiddens fine, somewhere else is not removing dot-files from the output when echoing.